src/routes/(root)/logout-everywhere/+page.server.js

```diff
@@ -0,0 +1,13 @@
+import { redirect } from '@sveltejs/kit';
```

Comment:
why can't we combine logout-everywhere and logout?

Reply:
It wasn't working, and it's is kind of experimental (for us) right now.